test/checkpoint/test_checkpoint.c

```diff
@@ -278,21 +281,33 @@ wt_connect(const char *config_open)
           g.checkpoint_slow_timing_stress ? "checkpoint_slow" : ""));
     }
 
+    /*
+     * Randomly decide on the eviction rate (fast or default).
+     */
+
+    if ((__wt_random(&rnd) % 15) % 2 == 0) {
+        fast_eviction_trigger = fast_eviction_target = true;
```

Comment:
If we are planning to go through having an option of doing `randomness`, it seems that having only one variable like `fast_eviction` is enough.

Also you can remove the braces for one line if statements

Reply:
Yep, thanks for pointing that out. I've changed it as you suggested.